src/main/java/com/microsoft/aad/msal4j/AuthorizationRequestUrlParameters.java

```diff
@@ -187,11 +199,20 @@ public Builder scopes(Set<String> val){
          * policy has not been met,{@link MsalServiceException} will contain claims that need be
          * consented to.
          */
-        public Builder claims(Set<String> val){
+        public Builder claims(String val){
```

Comment:
This is a breaking change, we would have to bump up major version.

Reply:
In the latest commit, I changed the original 'claims' parameter back to what it was and marked it as deprecated, and added a new 'claimsChallenge' parameter to handle all of the new behavior without being a breaking change.